src/components/ProfileBox/ProfileBox.module.scss

```diff
@@ -49,9 +49,8 @@
         justify-content: center;
         align-items: center;        
         // border: 1px solid rgba(255, 255, 255, 0.3);
```

Comment:
You can remove this commented code

Reply:
db138a252ab26d75a6f649041fff24356614ca05